gapis/api/gles/gles_pb/BUILD.bazel

```diff
@@ -35,4 +35,5 @@ go_proto_library(
     importpath = "github.com/google/gapid/gapis/api/gles/gles_pb",
     proto = ":gles_pb_proto",
     visibility = ["//visibility:public"],
+    deps = ["//gapis/memory/memory_pb:go_default_library"],  # keep
```

Comment:
I think the recommended approach here is to add a dummy import to the `docs.go` file instead of adding explicit `# keep` rules to the bazel rules.

Reply:
( but maybe this is different because of the proto aspect *shrug* )

Reply:
# keep is better, it just did not exist when I started so I did the doc.go trick instead
the advantage of the #keep it is probably easier to find non real dependancies and fix them up better in the future, the _ import might be because you actaully rely on the packages init functions being called (and also it will force them to be called even if not needed, preventing the compiler from removing the uneeded dependancy)

Reply:
yes, but don't, because there is an even better way coming next week that means you won't need either...
You can now write code generators that return both source and the deps the sources require, so the apic templates can carry the deps for you.

Reply:
bazelbuild/rules_go#1131 has most of that work, but it's not submitted yet